math/distributions.ts

```diff
@@ -0,0 +1,90 @@
+// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
+// This module is browser compatible.
+
+/** Math functions for generating distributions,
+ * @module
+ */
+
+const TWO_PI = Math.PI * 2;
+
+export interface RangeOptions {
+  /** Lower limit of range. */
+  min: number;
+  /** Upper limit of range. */
+  max: number;
+}
+
+const BASE_RANGE_OPTIONS: RangeOptions = {
+  min: 0,
+  max: 1,
+};
+
+/**
+ * Get a range of n evenly distributed numbers in ascending order.
+ * ```ts
+ * import { uniformRange } from "https://deno.land/std/math/distributions.ts";
+ *
+ * const range = uniformRange(100, {
+ *   min: 0,
+ *   max: 100,
+ * })
+ * ```
+ */
+export function uniformRange(
+  n: number,
+  options: RangeOptions = BASE_RANGE_OPTIONS,
+): number[] {
+  if (typeof n !== "number" || n <= 0 || !Number.isInteger(n)) return [];
```

Comment:
I think this should throw TypeError instead of returning empty array

Reply:
Made it throw a TypeError in the latest commit.